Sources/StreamChatUI/ChatMessageList/ChatMessage/Decorators/ChatThreadRepliesCountDecorationView.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatThreadRepliesCountDecorationView: ChatMessageDecorationView {
+    public var content: ChatMessage? {
+        didSet {
+            updateContentIfNeeded()
+        }
+    }
```

Comment:
Should we create as well a `ChatThreadRepliesCountDecorationViewContent`? 🤔 To allow us to include more data in the future, or maybe a bit overkill in this case? But imagine you want to have the `ChatChannel` data for some reason in the future.  (Just thinking out loud)

Reply:
It seems a bit too much to me. In any case, it would be composed in another view, so it should be fine.

Reply:
👍